cookbooks/app-react-bookmark-advanced/src/Root.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link, Outlet } from 'react-router-dom';
+import { Provider } from './Provider';
+import { BookmarkAppNavigation } from './BoomarkAppNavigation';
+import Create from './Create';
+
+export default function Root() {
```

Comment:
routes are not updated. How is this imported?

Reply:
what you mean, please explain?